settings.py

```diff
@@ -389,7 +389,8 @@
 
     'custom.icds_reports',
     'custom.enikshay.integrations.ninetyninedots',
-    'custom.pnlppgi'
+    'custom.pnlppgi',
+    'custom.enikshay'
```

Comment:
does this mean we can remove the `custom.enikshay.integrations.ninetyninedots` line above?